facia/app/controllers/FaciaController.scala

```diff
@@ -351,27 +352,58 @@ trait FaciaController
 
   def renderShowMore(path: String, collectionId: String): Action[AnyContent] =
     Action.async { implicit request =>
-      frontJsonFapi.get(path, fullRequestType).flatMap {
-        case Some(pressedPage) =>
-          val containers = Front.fromPressedPage(pressedPage, Edition(request), adFree = request.isAdFree).containers
-          val maybeResponse =
-            for {
-              (container, index) <- containers.zipWithIndex.find(_._1.dataId == collectionId)
-              containerLayout <- container.containerLayout
-            } yield {
-              val remainingCards: Seq[FaciaCardAndIndex] = containerLayout.remainingCards.map(_.withFromShowMore)
-              val adFreeFilteredCards: Seq[FaciaCardAndIndex] = if (request.isAdFree) {
-                remainingCards.filter(c => !checkIfPaid(c.item))
-              } else {
-                remainingCards
+      if (request.forceDCR) {
+        frontJsonFapi.get(path, fullDCRRequestType).flatMap {
+          case Some(pressedPage) =>
+            val matchedCollection = pressedPage.collections.find(_.id == collectionId)
+            // nb: length is an Option because not all types of pressedCollection have the property, but
+            //  pressedCollections coming from 'fullDCR' pressedPages *should* have the length prop. If they don't
+            //  then something has gone wrong. Narrowing the types and parsing the Collection
+            //  might be preferable to setting a default, but was deemed too time-consuming for
+            //  the time being.
+            matchedCollection
+              .map(collection => {
+
+                val liteCuratedLength = collection.liteCuratedLength.getOrElse(0)
+                val liteBackfillLength = collection.liteBackfillLength.getOrElse(0)
+
+                val pressedContent = collection.curated.takeRight(
+                  collection.curated.length - liteCuratedLength,
+                ) ++ collection.backfill.takeRight(collection.backfill.length - liteBackfillLength)
+                val startIndex = liteCuratedLength + liteBackfillLength
+
+                val config = collection.config
+
+                remoteRenderer.getCards(ws = ws, cards = pressedContent, startIndex = startIndex, config = config)
```

Comment:
What's the purpose of `startIndex` here, over say, just sending the latter half of the array?

Reply:
`startIndex` is used to generate the `data-link-name` for the cards, because this is how it's done in the existing Frontend implementation (link names are generated as though these cards are an extension of their parent container).

So the reason is just to replicate existing functionality, although @oliverlloyd [has also questioned](https://github.com/guardian/dotcom-rendering/pull/5116/files#r919897340) whether it's worth preserving this functionality

Reply:
My feeling here is that there is not a strong requirement to have exact position tracking for readers clicking on show more content. I would argue that it is _more_ beneficial to know that the card clicked on was part of the show more area so I suggest that we simplify the link-name attribute to 1) include the fact it is 'show-more' content and 2) have its own index, unrelated to the other cards in the container.

I suspect this content was indexed this way on Frontend because it just made sense to do it that way there but in the case of DCR, it is adding a lot of complexity and confusion which I don't think is justifiable.

In general, we should aim for parity; but not at all costs. It's a tricky balance but sometimes a feature or way of doing something doesn't make sense in DCR and is worth re-evaluating. Sometimes there is a better way. And sometimes the feature is entirely pointless and doesn't need to exist at all!

Reply:
I agree that (from the considerations I'm aware of) there doesn't seem to be a strong reason for the existing pattern. But I'm also conscious that I don't know what the original motivations were, and given that the `link-name` data is (I take it) consumed by people outside our team, it does seem like we should try to get an understanding of why this system is in place, and whether changing it will break anything, before changing the setup. Do we know who the stakeholders for this data are?

(The good news though is that it should be very easy to change the link name in DCR if we do decide to 👍 )

Reply:
Fair. We should inform them. I put a message in the Data channel letting them know

Reply:
Data replied. They are happy to have the index restarted at 1 🎉

Reply:
okay that sounds good. Happy to implement it, but would you object if we left this for a separate PR after the existing ones have been merged? It will be a fairly trivial code change on both Frontend and DCR, but as I'm in the process of splitting the existing PRs down, it would be good not to have to mess with them more than necessary at this point, as this change would touch a few different branches, until they're merged.